airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/integrations/destination_async/GlobalMemoryManager.java

```diff
@@ -88,8 +88,9 @@ public void free(final long bytes) {
     log.info("Freeing {} bytes..", bytes);
     currentMemoryBytes.addAndGet(-bytes);
 
-    if (currentMemoryBytes.get() < 0) {
-      log.warn("Freed more memory than allocated. This should never happen. Please report this bug.");
+    final long currentMemory = currentMemoryBytes.get();
+    if (currentMemory < 0) {
+      log.warn("Freed more memory than allocated ({})", currentMemory);
```

Comment:
```suggestion
      log.info("Freed more memory than allocated ({} of {})", bytes, currentMemory + bytes);
```

Reply:
Hmm... I think what I'll do because of codefreeze is bump the CDK versions but not publish a new connector version maybe?

Reply:
Oh... I guess because I'll be publishing the CDK, I *have* to merge this, huh

Reply:
yeah. Which... I'm oc anyway :P and I feel fine bypassing the freeze just to change a log message (:yolo:)

or we just leave this til post-freeze. It's not like this is causing major problems anyway